packages/cloudflare/src/cli/build/bundle-server.ts

```diff
@@ -33,7 +33,8 @@ export async function bundleServer(buildOpts: BuildOptions): Promise<void> {
   console.log(`\x1b[35m⚙️ Bundling the OpenNext server...\n\x1b[0m`);
 
   patches.patchWranglerDeps(buildOpts);
-  patches.updateWebpackChunksFile(buildOpts);
+  await patches.updateWebpackChunksFile(buildOpts);
+  await patches.patchVercelOgLibrary(buildOpts);
```

Comment:
It would be nice to be able to move the changes to `updateWorkerBundledCode()`
It would probably require adding externals to the ESBuild config.
With that we could `parse()` (ast-grep) the code once and apply multiple edits.

(This again could be handled in a follow-up  PR)

Reply:
I don't think we can do it there as this is patching a specific route's file whereas that function is patching handler.mjs, and the code for each of the routes doesn't appear to be bundled into the handler.mjs

Reply:
> the code for each of the routes doesn't appear to be bundled into the handler.mjs

yet :)

As I mentioned, we should move patches that add dependencies (`require`) from after the bundle to before/during the bundling.

I do have an ast-grep patch solving the route issue. I need to rebase and submit the PR.